cmd/catchpointdump/file.go

```diff
@@ -229,7 +234,13 @@ func loadCatchpointIntoDatabase(ctx context.Context, catchupAccessor ledger.Catc
 		}
 		if time.Since(lastProgressUpdate) > 50*time.Millisecond && catchpointFileSize > 0 {
 			lastProgressUpdate = time.Now()
-			printLoadCatchpointProgressLine(int(float64(progress)*50.0/float64(catchpointFileSize)), 50, int64(progress))
+			progressRatio := int(float64(progress) * barLength / float64(catchpointFileSize))
+			if progressRatio > barLength {
+				// gzip'ed file: adjust if read more than the estimated file size
+				catchpointFileSize = int64(progress)
```

Comment:
What is the purpose of updating `catchpointFileSize`?

Once this condition is hit, `catchpointFileSize` will every time be <= `progress` and the progress bar will always be at `barLength`. Is this correct?

Reply:
true, not really needed